### PR TITLE
Allow setting of name for custom license

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

File: cms/migrations/0017_auto__add_contentrepository.py [hunk 1]
```diff
@@ -85,6 +85,7 @@ def backwards(self, orm):
             u'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
             'last_author': ('django.db.models.fields.related.ForeignKey', [], {'blank': 'True', 'related_name': "'post_last_author'", 'null': 'True', 'to': u"orm['auth.User']"}),
             'localisation': ('django.db.models.fields.related.ForeignKey', [], {'to': u"orm['cms.Localisation']", 'null': 'True', 'blank': 'True'}),
+            'position': ('django.db.models.fields.PositiveIntegerField', [], {'default': '0', 'null': 'True', 'blank': 'True'}),
```
Comment (smn) [inline cms/migrations/0017_auto__add_contentrepository.py:88]: These got lost in the migration merge / rename around migration #16.

File: cms/migrations/0017_auto__del_field_contentrepository_license__add_field_contentrepository.py [hunk 2]
```diff
@@ -100,6 +100,7 @@ def backwards(self, orm):
             u'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
             'last_author': ('django.db.models.fields.related.ForeignKey', [], {'blank': 'True', 'related_name': "'post_last_author'", 'null': 'True', 'to': u"orm['auth.User']"}),
             'localisation': ('django.db.models.fields.related.ForeignKey', [], {'to': u"orm['cms.Localisation']", 'null': 'True', 'blank': 'True'}),
+            'position': ('django.db.models.fields.PositiveIntegerField', [], {'default': '0', 'null': 'True', 'blank': 'True'}),
```
Comment (smn) [inline cms/migrations/0017_auto__del_field_contentrepository_license__add_field_contentrepository.py:103]: These got lost in the migration merge / rename around migration #16.

**Comment by smn** [comment]
@miltontony ready for review.

**Comment by miltontony** [comment]
:+1:

**Pull request merged**